### DISCUSSION TITLE
Regression: POD no longer renders on GitHub

### DISCUSSION THREAD

**Discussion by zmughal** [community/community#63862] [Repositories]
### Select Topic Area

Bug

### Body

It appears that all POD no longer renders. This is also an issue when using POD as `README.pod`.

Example:

<https://github.com/Perl/perl5/blob/blead/Porting/release_schedule.pod>

It used to render: <https://web.archive.org/web/20221020105736/https://github.com/Perl/perl5/blob/blead/Porting/release_schedule.pod>

![image](https://github.com/github/markup/assets/94489/622948a5-02c3-4049-a48c-b3d847b7cac4)

and now it does not: <https://archive.is/AFAVz>.

![image](https://github.com/github/markup/assets/94489/b07006a5-7f32-4770-8a95-0ef0acb43edf)

Also submitted at <https://github.com/github/markup/issues/1715>.

**Comment by wchristian** [comment]
There's an update to this: https://github.com/github/markup/issues/1715#issuecomment-1710351574

**Comment by vaindil** [comment]
This should now be fixed, I apologize for the delay in getting this one resolved!
